service/src/test/java/uk/gov/hmcts/reform/fpl/model/DirectionResponseTest.java

```diff
@@ -0,0 +1,174 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+import java.util.UUID;
+
+import static java.util.Collections.singletonList;
+import static java.util.UUID.randomUUID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.DirectionAssignee.COURT;
+
+class DirectionResponseTest {
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    static {
+        mapper.findAndRegisterModules();
+    }
+
+    @Test
+    void shouldSerialiseRespondingOnBehalfOfToCorrectStringValueWhenRespondentValue() throws JsonProcessingException {
+        String serialised = mapper.writeValueAsString(DirectionResponse.builder()
+            .respondingOnBehalfOf("RESPONDENT_1")
+            .build());
+
+        assertThat(serialised).isEqualTo("{\"directionId\":null,\"assignee\":null,"
```

Comment:
This is bad. Pretty much unreadable IMO. Better to move it to fixture JSON files instead of keeping it as is.

Alternatively use tool that does JSON assertions (e.g. https://jsonassert.skyscreamer.org/cookbook.html) to compare only relevant parts of JSON. It will make it more readable in terms of what expected and more manageable at the same time because any additional field added to `DirectionResponse` won't break the test. Comparing JSON string 1-to-1 is quite painful.

Reply:
I like JSONAssert and have implemented where possible